sql/plan/alter_table.go

```diff
@@ -202,24 +198,32 @@ func (a *AddColumn) validateDefaultPosition(tblSch sql.Schema) error {
 	return nil
 }
 
-func (a *AddColumn) WithChildren(children ...sql.Node) (sql.Node, error) {
-	return NillaryWithChildren(a, children...)
+func (a AddColumn) WithChildren(children ...sql.Node) (sql.Node, error) {
```

Comment:
pointer receiver?

Reply:
drop and rename do the same thing, so i guess this is intentional?

Reply:
Yeah, this is a shortcut to dereferencing the pointer to make a copy (happens automatically)